components/compositing/windowing.rs

```diff
@@ -146,8 +147,15 @@ pub trait WindowMethods {
     /// will want to avoid blocking on UI events, and just
     /// run the event loop at the vsync interval.
     fn set_animation_state(&self, _state: AnimationState);
+    /// Get the animation state
+    fn get_animation_state(&self) -> AnimationState;
```

Comment:
I don't think this should be needed. The compositor doesn't need to know the animation state. See previous comment.

Reply:
If we move the heartbeat call, I think we can drop this.